cms/templatetags/cms_js_tags.py

```diff
@@ -48,12 +48,14 @@ def _render_plugin_js(plugin):
         try:
             child_classes = plugin_children[plugin.plugin_type]
         except KeyError:
-            child_classes = plugin.get_plugin_class().get_child_classes(slot=slot, page=page, instance=plugin)
+            plugin_class = plugin_pool.plugins[plugin.plugin_type]
```

Comment:
btw. in order to avoid this, I added a method

```
    @cached_property
    def plugin_class(self):
        return self.get_plugin_class()
```

to my model inheriting from ``CMSPlugin``. Later on, I forgot about it, hence your correct annotation 12 days ago about the ``AttributeError`` when attempting to access the ``plugin_class``. What about adding that method directly to ``CMSPlugin``?